CK3toEU4/Data_Files/blankMod/output/common/government_names/00_!ZZZ_added_government_names.txt

```diff
@@ -32,69 +32,99 @@ grand_duchy_monarchy = {
 	}
 }
 suffet = {
+	rank = {
+		1 = DUCHY
+		2 = KINGDOM
+		3 = EMPIRE
```

Comment:
This is meant to be Republic rather than Empire. I'll fix it in my files later

Reply:
Make sure you set it to being for republics then.

Reply:
I'll do that

Reply:
Alright, make sure that the government is a Republic as well.